xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -834,7 +834,8 @@
       "thumbnail": { "type": "string" },
       "lastplayed": { "type": "string" },
       "broadcastnow": { "$ref": "PVR.Details.Broadcast" },
-      "broadcastnext": { "$ref": "PVR.Details.Broadcast" }
+      "broadcastnext": { "$ref": "PVR.Details.Broadcast" },
+      "uniqueid": { "type": "integer" }
```

Comment:
Hum I don't know but you already have there a channel id defined that way:
`"channelid": { "$ref": "Library.Id", "required": true },`